frontend/apps/filemanager/filemanager.lua

```diff
@@ -1385,4 +1385,11 @@ function FileManager:onRefreshContent()
     self:onRefresh()
 end
 
+function FileManager:onMenuSearch()
+    if not self.ui then
+        self.menu:onShowMenu()
+    end
+    UIManager:broadcastEvent(Event:new("ShowMenuSearch"))
```

Comment:
`self.menu:handleEvent(...)` could probably work.
Or at least `UIManager:sendEvent()` as you do it below in common_settings_menu_table.lua.